docs/pages/setup/reference/backends.mdx

```diff
@@ -684,3 +684,52 @@ $ psql -d postgres
 postgres=# CREATE DATABASE teleport;
 postgres=# GRANT ALL PRIVILEGES ON DATABASE teleport TO dbuser;
 ```
+
+## SQLite
+
+The Auth Service uses the SQLite backend when no `type` is specified in the
+storage section in the Teleport configuration file, or when `type` is set to
+`sqlite` or `dir`. It's not designed for high throughput and it's not capable of
+serving the needs of Teleport's High Availability configurations; as such, it's
+only recommended for deployments of moderate size.
```

Comment:
To make the scaling recommendations more specific and actionable, what would you think of:

- Removing "; as such, it's
only recommended for deployments of moderate size"
- In the next paragraph, add a sentence similar to, "If you are planning to use SQLite as your backend, scale your cluster slowly and monitor the number of `SLOW TRANSACTION` logs".